<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Student Information System</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
  background-color: #0d6efd;
  color: #fff;
  font-weight: 600;
  font: 15pt;
  border-radius: 10px;
  border: 1px solid #0d6efd;
  padding: 5px 5px;
  outline: 0px;
cursor: pointer;
}
QPushButton:hover
 {
  background-color: rgb(98, 122, 255);
  border: 3px solid #9ac3fe;
cursor: pointer;
}

QLineEdit {
  border-radius: 8px;
  border: 1px solid #e0e4e7;
  padding: 5px 15px;
}

QLineEdit:focus {
  border: 1px solid #d0e3ff;
}

QLineEdit::placeholder {
  color: #767e89;
}

QLabel {
  color: #464d55;
  font-weight: 600;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>801</width>
      <height>631</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Student</string>
     </attribute>
     <widget class="QLineEdit" name="studentSearchLine">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>510</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 15pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Search </string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>321</width>
        <height>127</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 15pt &quot;Segoe UI&quot;;
</string>
         </property>
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="studentName">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 15pt &quot;Segoe UI&quot;;
border: 1px solid black;
</string>
         </property>
         <property name="placeholderText">
          <string>John</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font:15pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Gender:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="studentGender">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 15pt &quot;Segoe UI&quot;;
border: 1px solid black;
</string>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
         <item>
          <property name="text">
           <string>Male</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Female</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font:  15pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>ID-NO:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="studentId">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 15pt &quot;Segoe UI&quot;;
border: 1px solid black;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>2021-0534</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>40</y>
        <width>321</width>
        <height>137</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font:  15pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>CourseCode:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="studentCourse">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 15pt &quot;Segoe UI&quot;;
border: 1px solid black;
</string>
         </property>
         <property name="placeholderText">
          <string>BSCS</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font:  15pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Age:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="studentAge">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 15pt &quot;Segoe UI&quot;;
border: 1px solid black;
</string>
         </property>
         <property name="placeholderText">
          <string>20</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font:  15pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>YearLevel:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="studentYear">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 15pt &quot;Segoe UI&quot;;
border: 1px solid black;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>1st</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="studentAdd">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>180</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>ADD Student</string>
      </property>
     </widget>
     <widget class="QPushButton" name="studentEdit">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>180</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Edit Student</string>
      </property>
     </widget>
     <widget class="QPushButton" name="studentDelete">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>510</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Delete Student</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="studentStudent">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(98, 122, 255);</string>
         </property>
         <property name="text">
          <string>Student</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="studentCourses">
         <property name="text">
          <string>Courses</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="studentTable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>211</y>
        <width>741</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 15pt &quot;Segoe UI&quot;;</string>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>123</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID-NO</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Gender</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CourseCode</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Age</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>YearLevel</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Courses</string>
     </attribute>
     <widget class="QLineEdit" name="courseSearchLine">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>510</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 15pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="courseAdd">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>ADD Course</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>311</width>
        <height>90</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_10">
       <item row="0" column="0">
        <widget class="QLabel" name="label_19">
         <property name="styleSheet">
          <string notr="true">font: 15pt &quot;Segoe UI&quot;;
</string>
         </property>
         <property name="text">
          <string>CourseCode</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="courseCode">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 15pt &quot;Segoe UI&quot;;
border: 1px solid black;
</string>
         </property>
         <property name="placeholderText">
          <string extracomment="asdasd">BSCS</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_21">
         <property name="styleSheet">
          <string notr="true">font:  15pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>CourseName</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="courseName">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 15pt &quot;Segoe UI&quot;;
border: 1px solid black;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Computer Science</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="courseDelete">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>510</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Delete Course</string>
      </property>
     </widget>
     <widget class="QPushButton" name="courseEdit">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>160</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Edit Course</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="coursesStudent">
         <property name="text">
          <string>Student</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="coursesCourses">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(98, 122, 255);</string>
         </property>
         <property name="text">
          <string>Courses</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="courseTable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>741</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 15pt &quot;Segoe UI&quot;;</string>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>80</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>CourseCode</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CourseName</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
